xdsl/dialects/builtin.py

```diff
@@ -973,7 +987,7 @@ def from_region_or_ops(ops: List[Operation] | Region) -> ModuleOp:
 f64 = Float64Type()
 
 Builtin = Dialect(
-    [ModuleOp, UnregisteredOp],
+    [ModuleOp, UnregisteredOp, UnrealizedConversionCastOp],
```

Comment:
nit: add a comma at the end of this list so it's formatted as one item per row (making merges easier).